File: Basic Calculation of 2 Digits/co2d.java

//Basic Calculaton  of 2 Digits
import java.util.Scanner;
class so2d{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter 1st number : ");
        int n1=sc.nextInt();
        System.out.print("Enter 2nd number : ");
        int n2=sc.nextInt();
        sc.close();
        int sum=n1+n2;
        int diff=n1-n2;
        int quo=n1/n2;
        int pro=n1*n2;
        int rem=n1%n2;
        System.out.print("Summation : "+sum);
        System.out.print("Difference : "+diff);
        System.out.print("Product : "+pro);
        System.out.print("Quotient : "+quo);
        System.out.print("Reminder : "+rem);

    }
}

File: Basic Calculation of 2 Digits/co2d.cpp
//Basic Calculation of 2 digits
#include<iostream>
using namespace std;
void main(){
    int n1,n2,sum=0,diff=0,pro=0,rem=0;
    double quo=0.0;
    cout<<"Enter 1st number : ";
    cin>>n1;
    cout<<"Enter 2nd number : ";
    cin>>n2;
    sum=n1+n2;
    diff=n1-n2;
    pro=n1*n2;
    quo=n1/n2;
    rem=n1%n2;
    cout<<"Summation : "<<sum;
    cout<<"Difference : "<<diff;
    cout<<"Product : "<<pro;
    cout<<"Quotient : "<<quo;
    cout<<"Reminder : "<<rem;
}